manifests/backup/mysqldump.pp

```diff
@@ -24,12 +26,22 @@
     require       => Class['mysql::server::root_password'],
   }
 
-  mysql_grant { "${backupuser}@localhost/*.*":
-    ensure     => $ensure,
-    user       => "${backupuser}@localhost",
-    table      => '*.*',
-    privileges => [ 'SELECT', 'RELOAD', 'LOCK TABLES', 'SHOW VIEW', 'PROCESS' ],
-    require    => Mysql_user["${backupuser}@localhost"],
+  if $include_triggers {
```

Comment:
I'd have to say, I think I agree with "someone else" :-)

I think a select is good for times when you need to set a variable based on a number or string which could have a lot of different values - especially when you want to use a regular expression.  But it feels a little clunky for a boolean.  I've never seen a switch statement used on a boolean in other languages.

If this really needs to be a selector to be merged, so be it, but I'm not sure the code really benefits from the change in this case.